### PR TITLE
Building amp-html compliant static bloging site from local md files

### PR DESCRIPTION
**Pull request by des-des**

### CODE REVIEW

**Comment by nelsonic** [comment]
Wow! thanks @des-des :+1: 
I will read through after #24PullRequests :wink:

**Comment by Jbarget** [comment]
Good work team! @des-des @naazy @jackcarlisle @thegsi @Joshua-Ronan-Phillips @sohilpandya @mantagen

**Comment by nelsonic** [comment]
@des-des I have not forgotten about this... its just *massive* so I need to carve out a chunk of time to appreciate it. :wink:

File: package.json [hunk 1]
```diff
@@ -24,13 +26,20 @@
   },
   "homepage": "https://github.com/dwyl/adoro",
   "dependencies": {
+    "handlebars": "^4.0.5",
     "hapi": "^11.1.2",
     "inert": "^3.2.0",
-    "marked": "^0.3.5"
+    "marked": "^0.3.5",
```
Comment (nelsonic) [inline package.json]: We probably don't need *both* `marked` ***and*** `remarkable`, right...?

**Comment by nelsonic** [comment]
Nice one guys. clear code. decent tests functionality works as expected. merging. :+1:

File: package.json (hunk 1 above)
Comment (sohilpandya) [inline package.json]: nope. just remarkable. :)

**Pull request merged**